web/src/LdapEditPage.js

```diff
@@ -45,6 +47,7 @@ class LdapEditPage extends React.Component {
           this.setState({
             ldap: res.data,
           });
+          this.getCerts(res.data.owner);
```

Comment:
Can't it be optional?

Reply:
Yes, certificate is optional on UI and backend. 
This code only for  populate list of available certificates